app/models/manager_refresh/inventory_collection_default/infra_manager.rb

```diff
@@ -281,11 +281,47 @@ def hosts(extra_attributes = {})
         :builder_params              => {
           :ems_id => ->(persister) { persister.manager.id },
         },
+        :custom_reconnect_block      => reconnect_block
       }
 
       attributes.merge!(extra_attributes)
     end
 
+    def vms(extra_attributes = {})
```

Comment:
lets move this L284-L324 to https://github.com/Ladas/manageiq/blob/0815e847bd63282129eee90869bf9bf321c0224d/app/models/manager_refresh/inventory_collection_default.rb#L3

Reply:
@Ladas I did that but the specs started to fail for CloudManager so rolling back and keeping it on InfraManager level.

Reply:
ok, I will looks into that, let's start with infra then